docs/reference/sentiment.md

```diff
@@ -219,32 +219,35 @@ That's it! You have successfully built a Sentiment Analysis model that predicts
 
 ## Methods
 
-#### Initialize
+### ml5.sentiment()
+
+This method is used to load the sentiment model and store it in a variable. The ? means the argument is optional!
 
 ```js
-const sentiment = ml5.sentiment(model, ?callback);
+let sentiment = ml5.sentiment(model, ?callback);
 ```
 
 #### Parameters
 
-- **model**: REQUIRED. Defaults to 'moviereviews'. You can also use a path to a `manifest.json` file via a relative or absolute path.
-- **callback**: OPTIONAL. A callback function that is called once the model has loaded. If no callback is provided, it will return a promise that will be resolved once the model has loaded.
+- **model**: REQUIRED. Defaults to 'movieReviews'. You can also use a path to a `manifest.json` file via a relative or absolute path.
+- **callback(sentiment)**: Optional. A callback function that is called once the model has loaded. If no callback is provided, it will return a promise that will be resolved once the model has loaded.
 
 ---
 
-#### .predict()
+### sentiment.predict()
 
 > Given a number, will make magicSparkles
```

Comment:
This does not make a lot sense to me? Maybe we can remove this? Or Alan you have any specific reasons for keeping it?

Reply:
I did this to be consistent with the methods in the other models page. But I also realize that NN has a completely different structure compared to other models. Should we use the `.functionName()` style for all the pages?

Reply:
> I did this to be consistent with the methods in the other models page. But I also realize that NN has a completely different structure compared to other models. Should we use the `.functionName()` style for all the pages?

Oh no I refer to `> Given a number, will make magicSparkles` this line, maybe we should remove this line?

Reply:
Ohh got it. At least for this one, we should remove it. But the same question still, in NN a lot of `>` are working as method description. Should I reformat the whole thing now?

Reply:
> Ohh got it. At least for this one, we should remove it. But the same question still, in NN a lot of `>` are working as method description. Should I reformat that too? I might need to work on it tomorrow morning if that is ok :/

Yes, if time is permitted, let's remove `>` for NN as well. We can do a merge tomorrow morning around 10-11am ish before the presentation. If can't make it, it's fine, we can keep `>` for now.

Reply:
👌, that is feasible timing.